lib/layer.ts

```diff
@@ -99,14 +122,27 @@ export function getLayerARN(region: string, version: number, runtime: string) {
   return `arn:aws:lambda:${region}:${DD_ACCOUNT_ID}:layer:${layerName}:${version}`;
 }
 
+export function getExtensionLayerArn(region: string, version: number) {
+  const isGovCloud = region === "us-gov-east-1" || region === "us-gov-west-1";
+  if (isGovCloud) {
+    return `arn:aws-us-gov:lambda:${region}:${DD_GOV_ACCOUNT_ID}:layer:Datadog-Extension:${version}`;
+  }
+  return `arn:aws:lambda:${region}:${DD_ACCOUNT_ID}:layer:Datadog-Extension:${version}`;
+}
+
 export function getMissingLayerVersionErrorMsg(functionKey: string, formalRuntime: string, paramRuntime: string) {
   return (
     `Resource ${functionKey} has a ${formalRuntime} runtime, but no ${formalRuntime} Lambda Library version was provided. ` +
     `Please add the '${paramRuntime}LayerVersion' parameter for the Datadog serverless macro.`
   );
 }
 
-function generateLayerId(lambdaFunctionArn: string, runtime: string) {
+function generateLambdaLayerId(lambdaFunctionArn: string, runtime: string) {
```

Comment:
If we don't already have them, it might be nice to add some quick tests for `generateLambdaLayerId` and `generateExtensionLayerId`

Reply:
I tried to create unit tests for the generateLambdaLayerId, this was my approach, I create a sample lambda function, pass its functionArn and runtime to the generateLambdaLayerId function and since the Id is "-" delimited I can simply split and assert each part. However I cannot assert the part that was a hash of the function arn because the function arn is a token that is not available until synthesis of the cloud formation template. https://docs.aws.amazon.com/cdk/latest/guide/tokens.html 
![Screen Shot 2021-02-23 at 3 03 16 PM](https://user-images.githubusercontent.com/49878080/108919667-4545d900-75e8-11eb-9f6d-bf2830af37f9.png)

Reply:
I would just pass in a fake Arn as a string.

Reply:
Tests will be included in next commit